content/800-data-platform/100-accelerate/600-faq.mdx

```diff
@@ -33,11 +33,34 @@ This might be an issue with the TypeScript server not yet recognizing the regene
 
 VS Code should now recognize the `$extends` method.
 
-## What regions are Accelerate available in?
-
-Accelerate runs on Cloudflare's network. Cache hits are served from Cloudflare's 300+ locations. Cache misses can be served from 16 different locations at the time of writing. More will be added in the future.
-
-You can find the regions Accelerate is available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+## What regions are Accelerate's cache nodes available in?
+
+Accelerate runs on Cloudflare's network and cache hits are served from Cloudflare's 300+ locations. You can find the regions where Accelerate's cache nodes are available here: [https://www.cloudflare.com/network/](https://www.cloudflare.com/network/).
+
+## What regions is Accelerate's connection pool available in?
+
+When no cache strategy is specified or in the event of a cache miss, the Prisma Client query is routed through Accelerate's connection pool. Currently, queries can be routed through any chosen region among the 16 available locations.
```

Comment:
Also when the Accelerate extension is not used (as it is not needed for pure connection pooling needs)